embedded/sql/stmt.go

```diff
@@ -1119,6 +1119,10 @@ func (stmt *SelectStmt) compileUsing(e *Engine, implicitDB *Database, params map
 		return nil, ErrHavingClauseRequiresGroupClause
 	}
 
+	if len(stmt.groupBy) > 1 {
```

Comment:
If this check was not here, such situation would still be caught by comparing groupBy columns with orderBy columns (https://github.com/codenotary/immudb/pull/890/files#diff-1096943eee67a63ccd645a914b0b5c90a70bfb49378dbe325ead03929b68329aR39), right?

Reply:
yes, it'll be caught there before. But preferred to have an earlier validation